gradlew

```diff
@@ -51,7 +51,7 @@
 #       and GRADLE_OPTS) rely on word-splitting, this is performed explicitly;
 #       see the in-line comments for details.
 #
-#       There are tweaks for specific operating systems such as AIX, CygWin,
+#       There are tweaks for specific operating systems such as AIX, Cygwin,
```

Comment:
Brand

Reply:
yes, let's not change this

Reply:
By brand, I mean that https://www.cygwin.com/ writes their brand `Cygwin` not `CygWin`. That said. Dropped.